### PR TITLE
Create ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by wagnerand**
Looks like this:
____________________

### CODE REVIEW

**Comment by wagnerand** [comment]
Should this be added to `addons-server` as well? What about other repositories?

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+### Expected results:
+
+### Actual results:
+
+### Steps to reproduce the problem:
+
+### Link to the page you are looking at:
+
+### Screenshots and any relevant files (please attach or link to them):
```
Comment (EnTeQuAk) [inline ISSUE_TEMPLATE.md]: I would call that `Notes, screenshots and other relevant files (please attach or link to them)`

I feel like a separate section with notes of things that might or might not be relevant could be helpful. That's what QA is using too, e.g https://github.com/mozilla/addons-server/issues/3295

Comment (wagnerand) [inline ISSUE_TEMPLATE.md]: Done. Thanks!

**Comment by EnTeQuAk** [comment]
lgtm, r+

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+### Problem:
```
Comment (muffinresearch) [inline ISSUE_TEMPLATE.md]: I reckon you might be able to collapse this to more like 4 headings just in-case more headings make it appear daunting.

Here's a suggestion:

```md
### Describe the problem and steps to reproduce it.

### Actual results:

### Expected results: 

### Notes / URLS / Links to example add-ons or supplementary information:
```

Comment (wagnerand) [inline ISSUE_TEMPLATE.md]: My concern is the more we condense the more reporters will leave stuff out and we'll have to ask for it again.

Comment (muffinresearch) [inline ISSUE_TEMPLATE.md]: Yeah that's fair - it's trying to find a balance between a user being put off by so many questions and then not even filing versus requiring the necessary amount of information.

One thing is, if we err on the side of trying not to make the user feel overloaded with demands then at least we have the bug report and we can always ask for more information.

Comment (muffinresearch) [inline ISSUE_TEMPLATE.md]: As food for thought I quite like the way eslint document their bug reporting requirements: http://eslint.org/docs/developer-guide/contributing/reporting-bugs

Comment (wagnerand) [inline ISSUE_TEMPLATE.md]: @muffinresearch Thank you, I like that example! I update the template.

**Comment by muffinresearch** [comment]
Nice, looks good r+

**Pull request merged**